src/main/java/com/commercetools/project/sync/shoppinglist/ShoppingListSyncer.java

```diff
@@ -0,0 +1,101 @@
+package com.commercetools.project.sync.shoppinglist;
+
+import static com.commercetools.project.sync.util.SyncUtils.*;
```

Comment:
Beware of using wildcard. Sometimes Intellij automatically generated it.

Reply:
Yeah, I thought this would be corrected with spotlessCheck, but spotlessCheck doesn't update static imports. Corrected now, Thanks.